src/main/cache/issuesCache.ts

```diff
@@ -67,15 +67,20 @@ export class IssuesCache {
     }
 
     /**
-     * Get a scan results stored in the cache base on a given workspace and make sure the results are relevant.
-     * If the results are not relevant (the store interval period has passed) it will be removed them from the cache
+     * Get a scan results stored in the cache base on a given workspace and make sure the results are not expired.
+     * If the results are not expired (the store interval period has passed) it will be removed them from the cache
      * @param workspace - the workspace to search it's data
-     * @returns ScanResults if there are stored and relevant, undefined otherwise.
+     * @returns ScanResults if there are stored and not expired, undefined otherwise.
      */
-    public getOrClearIfNotRelevant(workspace: vscode.WorkspaceFolder): ScanResults | undefined {
+    public async getOrClearIfExpired(workspace: vscode.WorkspaceFolder): Promise<ScanResults | undefined> {
         let data: ScanResults | undefined = this.get(workspace);
         if (data && Utils.isIssueCacheIntervalPassed(data.oldestScanTimestamp)) {
             this.remove(workspace);
+            vscode.window.showInformationMessage("JFrog: Scan results for '" + workspace.name + "' have expired.", ...['Rescan']).then(answer => {
+                if (answer === 'Rescan') {
+                    vscode.commands.executeCommand('jfrog.xray.refresh');
```

Comment:
I think we should rename "jfrog.xray.refresh" to "jfrog.scan.refresh" in all code, since the scan now does more than just Xray scanning

Reply:
done